app/imports/ui/pages/CreateTestimony.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import '/client/style.css';
+import { Col, Container, Image, Row } from 'react-bootstrap';
+import { PAGE_IDS } from '../utilities/PageIDs';
+
+const CreateTestimony = () => (
+  <Container id={PAGE_IDS.CREATETESTIMONY} className="py-3">
```

Comment:
Should rename CREATETESTIMONY to CREATE_TESTIMONY for legibility because easier to read, and conforms the how other page IDs look too